android/sdl_android/src/main/java/com/smartdevicelink/managers/video/VideoStreamManager.java

```diff
@@ -284,25 +358,56 @@ public void onError(String info) {
      *                           works best for the currently connected module.
      * @param encrypted          a flag of if the stream should be encrypted. Only set if you have a supplied encryption library that the module can understand.
      */
-    public void startRemoteDisplayStream(Context context, Class<? extends SdlRemoteDisplay> remoteDisplayClass, VideoStreamingParameters parameters, final boolean encrypted) {
-        this.context = new WeakReference<>(context);
-        this.remoteDisplayClass = remoteDisplayClass;
-        int majorProtocolVersion = internalInterface.getProtocolVersion().getMajor();
+    @Deprecated
+    public void startRemoteDisplayStream(Context context, Class<? extends SdlRemoteDisplay> remoteDisplayClass, VideoStreamingParameters parameters, final boolean encrypted){
```

Comment:
Hello @kostyaBoss per this [comment](https://github.com/smartdevicelink/sdl_java_suite/pull/1434#discussion_r580557023)

If the developer uses this methos we should assume that the ranges are `null` and there for all capabilities would be supported.